examples/compose.js

```diff
@@ -0,0 +1,29 @@
+import dotty from 'dotty';
+
+const getDescriptorProps = (descriptorName, composables) => {
+  return composables.map(composable => {
+    return dotty.get(composable, 'compose.' + descriptorName);
+  });
+};
+
+function compose (...composables) {
+  const composable = {};
+  const methods = getDescriptorProps('methods', composables);
+
+  composable.compose = (...args) => {};
```

Comment:
Obviously need to implement this. Just trying to illustrate a more self-documenting way to handle the descriptor property mixing for now...